observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,57 @@
+Users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+[id="configuring-core-platform-monitoring-first-steps_{context}"]
+== Configuring core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include setting up storage and configuring options for Prometheus, Alertmanager, and other monitoring components.
+
+[NOTE]
+====
+By default, in a newly installed {product-title} system, users can query and view collected metrics.
+You need only configure an alert receiver if you want users to receive alert notifications.
+Any other configuration options listed here are optional.
+====
+
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack[Create the `cluster-monitoring-config` `ConfigMap` object] if it does not exist.
+* xref:../../observability/monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[Configure alert receivers] so that Alertmanager can send alerts to an external notification system such as email, Slack, or PagerDuty.
+* For shorter term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-persistent-storage_configuring-the-monitoring-stack[configure persistent storage] for Prometheus and Alertmanager to store metrics and alert data.
+Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[NOTE]
+====
+By default, in a newly installed {product-title} system, the monitoring `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
+====
++
+* For longer term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring_remote_write_storage_configuring-the-monitoring-stack[configure the remote write feature] to enable Prometheus to send ingested metrics to remote systems for storage.
++
+[IMPORTANT]
+====
+Be sure to xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#adding-cluster-id-labels-to-metrics_configuring-the-monitoring-stack[add cluster ID labels to metrics] for use with your remote write storage configuration.
+====
++
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#assigning-tolerations-to-monitoring-components_configuring-the-monitoring-stack[Assign tolerations] to monitoring stack components to enable moving them to tainted nodes.
```

Comment:
I think this reads a bit easier, though I'm not sure if it is technically correct:

```suggestion
* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#assigning-tolerations-to-monitoring-components_configuring-the-monitoring-stack[Assign tolerations] to monitoring stack components so that administrators can move them to tainted nodes.
```

Reply:
Changed per your suggestion